src/Servers/Kestrel/Core/src/Internal/KestrelServerImpl.cs

```diff
@@ -183,7 +190,8 @@ async Task OnBind(ListenOptions options, CancellationToken onBindCancellationTok
                 // Quic isn't registered if it's not supported, throw if we can't fall back to 1 or 2
                 if (hasHttp3 && _multiplexedTransportFactories.Count == 0 && !(hasHttp1 || hasHttp2))
                 {
-                    throw new InvalidOperationException("This platform doesn't support QUIC or HTTP/3.");
+                    // TODO (acasey): is this good enough of should we track whether UseQuic was called?
+                    throw new InvalidOperationException("This platform doesn't support QUIC or HTTP/3.  Alternatively, you may need to call `UseQuic` on `IWebHostBuilder` to enable it");
```

Comment:
```suggestion
                    throw new InvalidOperationException("Unable to bind an HTTP/3 endpoint. This could be because QUIC has not been configured using UseQuic, or the platform doesn't support QUIC or HTTP/3.");
```
It would be nice if there was a way to figure out if UseQuic has been called, but it isn't supported, to provide a more useful error message here.